packages/components/src/Tile/TileGrid/_docs/TileGrid.stories.tsx

```diff
@@ -78,3 +78,85 @@ export const FlipOneNotOthers: Story = {
     })
   },
 }
+
+// @Zystix: Add Chromatic flag to tests or add to stickersheet.
+
+// Does it work with one tile
```

Comment:
Maybe just clean up by removing the comments (and L107). Code should be self-documenting, so usually comments are reserved for business logic/hard to read code (which may be caused by consuming a library) :)